lib/uo_cli.rb

```diff
@@ -438,6 +464,8 @@ def self.create_project_folder(dir_name, empty_folder = false, overwrite_project
       default_post_processor = URBANopt::Scenario::ScenarioDefaultPostProcessor.new(run_func)
       scenario_report = default_post_processor.run
       scenario_report.save
+      scenario_report.feature_reports.each(&:save_feature_report)
```

Comment:
@vtnate I have a merged conflict with develop, I saw https://github.com/urbanopt/urbanopt-cli/blob/2cd611595c0f56ce120afcc258da06bd17f8048b/lib/uo_cli.rb#L441 this is added. Should I add this to resolve the conflict?

Reply:
Yes, please do. I just merged that into develop yesterday. Thanks!